runway/hooks/staticsite/auth_at_edge/domain_updater.py

```diff
@@ -0,0 +1,99 @@
+"""User Pool Client Domain Updater."""
+import logging
+# pylint: disable=unused-import
+from typing import Any, Dict, Optional, Union
+
+import boto3
+
+from ....cfngin.session_cache import get_session
+# pylint: disable=unused-import
```

Comment:
this should be in-line so it does not apply to the entire file, only a single line

Reply:
What's the right way to get this one working? I've got a catch-22 where if I do the consolidated style of typing it gives me a line that's to long, but if I leave it as it is now I can't add the pylint call inline:

`# pylint: disable=unused-argument
def update(context,  # type: context.Context
           provider,  # type: BaseProvider
           **kwargs  # type: Optional[Dict[str, Any]]
          ):  # noqa: E124
    # type: (...) -> Union[Dict[str, Any], bool]`